css/design_system/utils/text.scss

```diff
@@ -0,0 +1,82 @@
+@mixin docstring($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">Example text</p>
+   */
+}
+
+@mixin docstring_messy($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">exaMple tEXT</p>
+   */
+}
+
+// text-align
+.s-text-start {
+  @include docstring("s-text-start", "text-align");
+  text-align: start;
+}
+
+.s-text-center {
+  @include docstring("s-text-center", "text-align");
+  text-align: center;
+}
+
+.s-text-end {
+  @include docstring("s-text-end", "text-align");
+  text-align: end;
+}
+
+// text-decoration
+.s-text-decor-line-through {
+  @include docstring("s-text-decor-line-through", "text-decoration");
+  text-decoration: line-through;
+}
+
+.s-text-decor-none {
+  @include docstring("s-text-decor-none", "text-decoration");
+  text-decoration: none;
+}
+
+.s-text-decor-underline {
+  @include docstring("s-text-decor-underline", "text-decoration");
+  text-decoration: underline;
+}
+
+// text-transform
+.s-text-capitalize {
+  @include docstring_messy("s-text-capitalize", "text-transform");
+  text-transform: capitalize;
+}
+
+.s-text-lowercase {
+  @include docstring_messy("s-text-lowercase", "text-transform");
+  text-transform: lowercase;
+}
+
+.s-text-uppercase {
+  @include docstring_messy("s-text-uppercase", "text-transform");
+  text-transform: uppercase;
+}
+
+// font-weight
+$font-weights-options: "ultra-light", "light", "normal", "bold", "heavy";
+
+@each $option in $font-weights-options {
+  $class-name: "s-fw-#{$option}";
```

Comment:
I have a slight preference for `s-font-*` as the prefix, just because it's a little easier to read/remember. What do you think?

Reply:
Good with me, I was iffy on that one.